kafka-ui-react-app/src/components/Brokers/Brokers.tsx

```diff
@@ -98,15 +98,21 @@ const Brokers: React.FC = () => {
           </tr>
         </thead>
         <tbody>
-          {diskUsage?.map(({ brokerId, segmentSize, segmentCount }) => (
-            <tr key={brokerId}>
-              <td>{brokerId}</td>
-              <td>
-                <BytesFormatted value={segmentSize} />
-              </td>
-              <td>{segmentCount}</td>
+          {diskUsage && diskUsage.length !== 0 ? (
+            diskUsage.map(({ brokerId, segmentSize, segmentCount }) => (
```

Comment:
why is it better than `diskUsage?.map`?

Reply:
Do we really need '?' if we had already checked that diskUsage was not udefined? 
Here is the check
`diskUsage && diskUsage.length`